### DISCUSSION TITLE
Audio delay in sensible soccer

### DISCUSSION THREAD

**Discussion by amyren1966** [BlitterStudio/amiberry#868] [Q&A] [answered]
Are there any settings that I can use to reduce the delay in sound effects for games like sensible soccer.
When I play Sensible Soccer v1.2 I notice that the sounds are several seconds behind whats going on.
For exampe when the game is over, the players have almost left the area before the sound of the whiste comes.
This is v.4.1.6, set up as a A1200, loading the game from floppy.

**Comment by midwan** [comment, marked as answer]
Yes, try reducing the size of the audio buffer.

**Reply by solskogen** [reply]
And/or change from push to pull (audio tab)

**Reply by amyren1966** [reply]
I tried to change it to pull, but did not notice any difference. Only tested briefly, I might try again tomorrow.

**Reply by amyren1966** [reply]
just discovered a strange bug when messing with those audio settings. I will post it in the issues section.

**Comment by amyren1966** [comment]
Yes, that helped. I set the buffer to minimum. They still starts to leave a short while before the whistle, but it is better.

**Comment by amyren1966** [comment]
Testing the latest builds, 4.2 and 4.3beta on 32 bit Raspberry Pi OS I now got Sensible Soccer Int. Ed 1.2 to play without noticing any audio delays. I had the sound settings set to pull and buffer = 1.
Also tried build 4.2 on a Manjaro 64bit setup, but then the same game setting caused a major slowdown. Setting the buffersize to 5, and the game ran at normal speed with audio synced.

**Reply by midwan** [reply]
In my tests on Manjaro, I can't set the buffer size to less than 6 when using Pull mode, otherwise I will experience some distortions in some cases. But using that configuration, I don't have any lag or other issues (so far).
